cmd/batch-handlers.go

```diff
@@ -570,7 +571,7 @@ func (r *BatchJobReplicateV1) StartFromSource(ctx context.Context, api ObjectLay
 				ri.trackCurrentBucketObject(r.Target.Bucket, oi, success)
 				globalBatchJobsMetrics.save(job.ID, ri)
 				// persist in-memory state to disk after every 10secs.
-				logger.LogIf(ctx, ri.updateAfter(ctx, api, 10*time.Second, job.Location))
+				logger.LogIf(ctx, ri.updateAfter(ctx, api, 10*time.Second, job.Location, job.Type()))
```

Comment:
```suggestion
				logger.LogIf(ctx, ri.updateAfter(ctx, api, 10*time.Second, job))
```

We could pass the `job` directly?